src/main/java/br/eti/kinoshita/testlinkjavaapi/constants/TestLinkMethods.java

```diff
@@ -29,60 +29,65 @@
-                                                                                                                                                                                                                                                                                                                                                                                                    "tl.getExecCountersByBuild"), UPDATE_TEST_CASE_CUSTOM_FIELD_VALUE(
-                                                                                                                                                                                                                                                                                                                                                                                                            "tl.updateTestCaseCustomFieldDesignValue"), SET_TEST_CASE_EXECUTION_TYPE(
-                                                                                                                                                                                                                                                                                                                                                                                                                    "tl.setTestCaseExecutionType"), GET_PROJECT_PLATFORMS(
-                                                                                                                                                                                                                                                                                                                                                                                                                            "tl.getProjectPlatforms"), REMOVE_PLATFORM_FROM_TEST_PLAN(
-                                                                                                                                                                                                                                                                                                                                                                                                                                    "tl.removePlatformFromTestPlan"), ADD_PLATFORM_TO_TEST_PLAN(
-                                                                                                                                                                                                                                                                                                                                                                                                                                            "tl.addPlatformToTestPlan");

```

Comment:
Yuck. I hadn't notice it got all that messed up. Thakns for fixing the formatting here.